src/main/webapp/js_tests/paginationScript.test.js

```diff
@@ -0,0 +1,90 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const TOTAL_CARDS_TO_DISPLAY = 3;
+const MAX_LIST_VIEW_NUMBER = 15;
+
+let testResultsContent = [];
+let testResultsCardsList = [];
+
+for (let i = 0; i < 6; i++) {
```

Comment:
Why 6?

Reply:
I'm just filling the testResultsCardList with 6 cards, enough to test what I need it to.